scripts/with_pygithub/notifier.py

```diff
@@ -0,0 +1,31 @@
+import argparse
+from github import Github
+
+DOC_LABEL = "to be documented"
+DOC_MSG = "This message was sent automatically." # TODO change it
+
+
+if __name__ == '__main__':
+    print("logs: start")
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--token", type=str, required=True)
+    parser.add_argument("--pr_id", type=int, required=True)
+    parser.add_argument("--event", type=str, required=True)
+    parser.add_argument("--label", type=str)
+    parser.add_argument("--repo", type=str)
```

Comment:
Still unresolved

Reply:
"I can" does not mean "I should" :) 
But, I tried, and find out that I can't do it easy :(
I created a new folder "with_pygithub", because the `import github` imports file `scripts\github`, not `pygithub` module. Importing `common` that is in the parent directory is difficult, and the code will be less readable (see https://stackoverflow.com/questions/16981921/relative-imports-in-python-3). 
Just `import ..common` doesn't work here. 
Maybe passing a repository as a parameter is not so bad solution?

Reply:
The straightforward solution is to rename our custom `github.py` to resolve the name clash. Moving to a different folder and using relative imports solely for resolving the name collision looks overengineered. After all, there's

```
# TODO: think about using library for GitHub API
```

... in `github.py` so if we are willing to use `PyGithub` as such a library we have to rename our custom file anyway.